### PR TITLE
Add compas_slicer (Slicing package for COMPAS framework)

### PR DESCRIPTION
**Pull request by gonzalocasas**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gonzalocasas** [comment]
@conda-forge-admin please re-render

**Comment by gonzalocasas** [comment]
@conda-forge-admin, please rerender

**Comment by gonzalocasas** [comment]
This is ready for review. @conda-forge-admin, please ping team

**Comment by gonzalocasas** [comment]
@joburger @ioannaMitropoulou could you please confirm you agree to be maintainers of this recipe? thx!

**Comment by joburger** [comment]
Hereby my confirmation that I agree to be a maintainer of this recipe!

File: recipes/compas_slicer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/compas_slicer-{{ version }}.tar.gz
+  sha256: 6d7cc6c1a84ac28f32b33b8ad6d78d1a3f7e7b024093dd4a156594c13f428c4b
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/compas_slicer/meta.yaml:15]: I suspect that this one can be `noarch: python`. Can you check if there are compiled extensions of something that is python version/platform specific that would prevent us from using noarch here?

Comment (gonzalocasas) [reply recipes/compas_slicer/meta.yaml:16]: you're totally right! updated it to `noarch`

**Pull request merged**